internal/util/importutilv2/csv/row_parser.go

```diff
@@ -0,0 +1,438 @@
+package csv
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/milvus-io/milvus-proto/go-api/v2/schemapb"
+	"github.com/milvus-io/milvus/pkg/util/merr"
+	"github.com/milvus-io/milvus/pkg/util/typeutil"
+	"github.com/samber/lo"
+)
+
+type RowParser interface {
+	Parse(raw []string) (Row, error)
+}
+type rowParser struct {
+	dim          int
```

Comment:
It seems like it can't handle with multiple vector?

Reply:
Yes, I missed that. I will refer to the solution in json import and support multiple vectors by changing dim to id2Dim, and add corresponding test cases in the tests/integration/import folder.